skimage/__init__.py

```diff
@@ -68,10 +68,35 @@
 
 """
 
-import sys
+__version__ = '0.19.0.dev0'
 
+__all__ = [
+    'color',
+    'data',
+    'draw',
+    'exposure',
+    'external',
+    'feature',
+    'filters',
+    'future',
+    'graph',
+    'io',
+    'measure',
+    'metrics',
+    'morphology',
+    'registration',
+    'restoration',
+    'segmentation',
+    '_shared',
+    'transform',
+    'util',
+    'viewer'
+]
+
+from .util.lazy import install_lazy
+__getattr__, __dir__, _ = install_lazy(__name__, __all__)
 
-__version__ = '0.19.0.dev0'
+import sys
```

Comment:
Is `import sys` needed here? I don't see any explicit usage of it.

Reply:
Moved to where it is used.